tinytuya/BulbDevice.py

```diff
@@ -466,17 +446,29 @@ def set_white(self, brightness=-1, colourtemp=-1, nowait=False):
                 "set_white: The colour temperature needs to be between 0 and 1000.",
             )
 
-        payload = self.generate_payload(
-            CONTROL,
-            {
-                self.DPS_INDEX_MODE[self.bulb_type]: self.DPS_MODE_WHITE,
-                self.DPS_INDEX_BRIGHTNESS[self.bulb_type]: brightness,
-                self.DPS_INDEX_COLOURTEMP[self.bulb_type]: colourtemp,
-            },
-        )
+        payload = {
+            self.DPS_INDEX_BRIGHTNESS[self.bulb_type]: brightness,
+            self.DPS_INDEX_COLOURTEMP[self.bulb_type]: colourtemp,
+        }

```

Comment:
Instead of `payload` should we move to `dps_values` or something like that to not confuse this with raw "payloads"?